core/controllers/base.py

```diff
@@ -425,7 +459,13 @@ def _render_exception(self, error_code, values):
                 self.render_template('pages/error/error.html')
 
     def handle_exception(self, exception, unused_debug_mode):
-        """Overwrites the default exception handler."""
+        """Overwrites the default exception handler
+
+        Args:
+            exception: int. Raised while assertion of error code
```

Comment:
Oh. I thought exception here was an error code just like 404,401 etc. Made changes here. Kindly look into it